### PR TITLE
Remove sticks arming

### LINKED ISSUE #4583: Remove support for stick arming
Stick arming comes from the ancient era of RC systems where we didn't have enough switches and sometimes we had no switches at all.

This is no longer the case - even the cheapest RC systems have at least 6 channels (more often more than 8) where you can dedicate a channel to arming.

Stick arming brings along a bunch of hacks to the code to accommodate for different behavior at zero throttle. Sometimes it causes unexpected behavior - i.e. trying to do a full  yaw at zero throttle. And if combined with MOTOR_STOP this is becomes even less safe.

### PR DESCRIPTION
**Pull request by DzikuVx**
closes #4583

### CODE REVIEW

**Comment by DzikuVx** [comment]
ok, looks like I have to fix conflicts :)

File: src/main/fc/rc_modes.c [hunk 1]
```diff
@@ -54,11 +54,6 @@ boxBitmask_t rcModeActivationMask; // one bit per mode defined in boxId_e
 PG_REGISTER_ARRAY(modeActivationCondition_t, MAX_MODE_ACTIVATION_CONDITION_COUNT, modeActivationConditions, PG_MODE_ACTIVATION_PROFILE, 0);
 PG_REGISTER(modeActivationOperatorConfig_t, modeActivationOperatorConfig, PG_MODE_ACTIVATION_OPERATOR_CONFIG, 0);
 
-bool isUsingSticksForArming(void)
-{
-    return isUsingSticksToArm;
```
Comment (digitalentity) [inline src/main/fc/rc_modes.c:59]: @DzikuVx you missed the declaration of `isUsingSticksToArm` and other places where it's referenced.

**Review by digitalentity — CHANGES_REQUESTED** [review]

**Comment by DzikuVx** [comment]
@digitalentity rebased and fixed

**Review by digitalentity — APPROVED** [review]
LGTM

**Comment by hali9** [comment]
If I good undestand code this PR also removed fixed_wing_auto_arm
This is probably not intentional.

**Comment by oldgeezy** [comment]
So did this happen?  Because I can't arm with sticks anymore.  And I can't find the setting to prevent disarm unless the throttle is idle.

**Pull request merged**